implementations/rust/ockam/ockam_api/src/cloud/project.rs

```diff
@@ -446,7 +446,7 @@ mod tests {
                 authority_identity: bool::arbitrary(g)
                     .then(|| hex::encode(<Vec<u8>>::arbitrary(g))),
                 okta_config: bool::arbitrary(g).then(|| OktaConfig::arbitrary(g)),
-                confluent_config: bool::arbitrary(g).then(|| ConfluentConfigResponse::arbitrary(g)),
+                confluent_config: bool::arbitrary(g).then(|| KafkaConfigResponse::arbitrary(g)),
```

Comment:
That's for generating test data, so it get the same answer as [this question](https://github.com/build-trust/ockam/pull/5930#discussion_r1390603096). The real question is "why is not `ConfluentConfig` instead of `ConfluentConfigResponse`?". (I've changed that in my current PR)